### PR TITLE
Take CoinFi News from private beta to public launch

### PR DESCRIPTION
**Pull request by hanchang**
https://app.asana.com/0/683768070860536/823481256129514

### CODE REVIEW

**Comment by jessehon** [comment]
Looks like theres an error rendering this

image

File: app/javascript/App/components/CoinList/CoinListDrawer.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from 'react'
```
Comment (anthotsang) [inline app/javascript/App/components/CoinList/CoinListDrawer.js]: Seems like a lot of the old App/components are resurrected. Is this on purpose?

File: app/javascript/App/bundles/common/components/CoinList.tsx [hunk 2]
```diff
@@ -11,17 +12,26 @@ interface Props {
...
-        onSelectCoin={props.onSelectCoin}
-        isSelected={
-          !!props.selectedCoinSlug && props.selectedCoinSlug === coin.slug
-        }
-      />
-    ))}
+    {!props.loggedIn && props.isWatchlist ? (
+      <div className="pa3 tc">
+        Sign up to see coins on your Watchlist here.
+        <a className="btn btn-md btn-blue mt3" href="/register">
```
Comment (anthotsang) [inline app/javascript/App/bundles/common/components/CoinList.tsx]: You added a url constant but don't use it :sweat_smile:

File: app/javascript/App/bundles/NewsfeedPage/NewsBody.tsx [hunk 3]
```diff
@@ -104,12 +104,12 @@ export default class NewsBody extends React.Component<Props, State> {
             rel="nofollow"
             className="break-all"
           >
-            <Icon name="link" className="mr1 f7" regular={true} />
+            <Icon name="link" className="mr1 f7" styleType="regular" />
```
Comment (anthotsang) [inline app/javascript/App/bundles/NewsfeedPage/NewsBody.tsx]: Given the icon component logic, it seems you might as well remove styleType="regular"

**Pull request merged**